Jenkinsfile

```diff
@@ -42,6 +42,11 @@ node(workerLabel) {
         } // end docker
       } // end if
       if (scmVars.GIT_LOCAL_BRANCH ==~ /master|develop/) {
+        docker_push_tags = [scmVars.GIT_LOCAL_BRANCH, 'latest']
```

Comment:
We want to assign `latest` tag for images from `master` branch and `develop` tag for `develop` branch